pkg/ibmcsidriver/controller.go

```diff
@@ -313,14 +256,21 @@ func (csiCS *CSIControllerServer) ValidateVolumeCapabilities(ctx context.Context
 		return nil, commonError.GetCSIError(ctxLogger, commonError.EmptyVolumeID, requestID, nil)
 	}
 
+	//Volume ID is in format volumeID:volumeAccessPointID
+	tokens := strings.Split(volumeID, ":")
+	if len(tokens) != 2 {
+		ctxLogger.Info("CSIControllerServer-ValidateVolumeCapabilities...", zap.Reflect("Volume ID is not in format volumeID:accesspointID", tokens))
+		return nil, commonError.GetCSIError(ctxLogger, commonError.InternalError, requestID, nil)
```

Comment:
pl check if we want to return as InternalError or specific error so that user can take action